.github/workflows/docker-compose-test.yml

```diff
@@ -0,0 +1,48 @@
+name: Docker compose test
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Deploy Fedora Pod
+        uses: actions-hub/kubectl@master
+        env:
+          KUBE_CONFIG: ${{ secrets.HETZNER_KUBE_CONFIG }}
+        with:
+          args: apply -f docker-compose/podman.yaml
+
+      - name: Wait for pod to be ready
+        uses: actions-hub/kubectl@master
+        env:
+          KUBE_CONFIG: ${{ secrets.HETZNER_KUBE_CONFIG }}
+        with:
+          args: wait --for=condition=ready pod -l app=podman-priv --timeout=120s
```

Comment:
doesn't this wait only for the host container to get started (not the docker-compose stack)?